src/server/api/enums/ClipboardContentTypes.ts

```diff
@@ -0,0 +1,5 @@
+enum ClipboardContentTypes {
```

Comment:
I'm not sure if this is standard practice in TypeScript world, but I prefer singular names for `enums`

```suggestion
enum ClipboardContentType {
```

Reply:
@simonhamp do you still want this enum to have a singular name even though ClipboardEvents stays plural?